### PR TITLE
Html things

### PR DESCRIPTION
**Pull request by joshuajhun**
So I added an h1 for our application. 💯

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>This sweet web app.</title>
+  </head>
+  <body>
+    <h1> Some great title</h1>
```
Comment (martensonbj) [inline index.html]: YESSSSSS this is what I wanted. Can you also add a <p> tag??

**Comment by martensonbj** [comment]
Hey made a suggestion on the code request. Check it out.

File: index.html (hunk 1 above)
Comment (joshuajhun) [inline index.html]: Ok. I can totally do that. I got you fam 💯

**Comment by martensonbj** [comment]
👍 Love it you rock.

**Pull request merged**